native/cocos/renderer/gfx-gles3/GLES3PipelineCache.cpp

```diff
@@ -33,25 +33,51 @@
 
 namespace cc::gfx {
 
+//#define PIPELINE_CACHE_FORCE_INCREMENTAL
+
+#if defined(_WIN32) && !defined(PIPELINE_CACHE_FORCE_INCREMENTAL)
+#define PIPELINE_CACHE_FULL
+#else
+#define PIPELINE_CACHE_INCREMENTAL
+#endif
+
 namespace {
 const char *fileName = "/pipeline_cache_gles3.bin";
```

Comment:
Will this kind of path working on all platforms?

Reply:
Incremental and full are supported on all platforms, and here we are controlling the default behavior.